src/System.Management.Automation/engine/PSVersionInfo.cs

```diff
@@ -316,6 +316,43 @@ internal static SemanticVersion PSV6Version
     }
 
     /// <summary>
+    /// Represents an implementation of '$PSVersionTable' variable.
+    /// The implementation contains ordered 'Keys' and 'GetEnumerator' to get user-friendly output.
+    /// </summary>
+    public sealed class PSVersionHashTable : Hashtable, IEnumerable
```

Comment:
VS Code detects this as error and build failed too without explicity adding `IEnumerable`.